libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/ConwayFeatures.hs

```diff
@@ -399,19 +403,25 @@ testGov pf = do
               )
             ]
         )
-    -- Wait two epochs
+    -- Wait three epochs
     eitherEpochState1 = runEPOCH (EPOCH pf) epochState0 (EpochNo 2) poolDistr
   epochState1 <- expectRight "Error running runEPOCH: " eitherEpochState1
   let
     constitution = epochState1 ^. esLStateL . lsUTxOStateL . utxosGovStateL . cgEnactStateL . ensConstitutionL
   assertExprEqualWithMessage "constitution after one epoch" constitution def
+  epochState1' <-
```

Comment:
Same here could you avoid mixing ticks with numbers. It is hard to follow the dependency order